requirements.txt

```diff
@@ -19,5 +19,5 @@ scipy
 sounddevice
 torch
 tqdm
-wavio<=0.0.4  # Breaking change in 0.0.5
+wavio>=0.0.5  # Breaking change with older versions
```

Comment:
Nit: since the code is backward-compatible, no need to constrain to `>=0.0.5`.

Reply:
I think so too, but now I am a bit confused if I also think about your comments re the environment_x.yml files. Are these treated differently? Should I keep the constraints there but remove them from requirements.txt?